mscore/scoreview.cpp

```diff
@@ -5888,6 +5888,16 @@ void ScoreView::cmdAddRemoveBreaks()
                         }
                   }
             }
+      // deal with last measure of selection, but skip last measure of score
+      if (count == interval && endMeasure != endMeasure->system()->lastMeasure()) {
```

Comment:
Hmm, true. I tried a different loop condition before, but couldn't get it to work, let's try this then...